.github/workflows/labeling.yml

```diff
@@ -14,5 +19,3 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           label-pattern: '- \[(.*?)\] ?`(.+?)`'
-          # Label issues from last week
-          offset: '7d'
```

Comment:
What's the default offset?

Reply:
The default offset is one month, but this argument is only valid for the schedule event.

Reply:
Ah, I see. So this will just cause autolabelling happen only as things are opened or edited. Is that sufficient to capture all events that would cause us to change labels?

Reply:
Yes because only `opened` and `edited` events updates an issue (or PR) description.